app/time_tracking/work_time/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.utils.timesince import timesince
+import time
+
+class WorkTime(models.Model):
+    start_time = models.DateTimeField(blank=False, null=False)
+    end_time   = models.DateTimeField(blank=True, null=True)
+
+    work_time  = models.PositiveIntegerField(blank=True, null=True, editable=False)
+
+    start_unix_time = models.PositiveIntegerField(blank=False, null=False, editable=False)
+    end_unix_time = models.PositiveIntegerField(blank=True, null=True, editable=False)
+
+    days_count = models.PositiveIntegerField(blank=False, null=False, editable=False)
+
+    owner = models.ForeignKey(
+        User,
+        limit_choices_to={'is_staff': False},
+        on_delete=models.CASCADE,
+        related_name='work_times',
+    )
+
+    def save(self, *args, **kwargs):
+        """ On create, set start_time """
+        if not self.id:
+            time_now = timezone.now()
+            self.start_time = time_now
```

Comment:
from design perspective, `start_time` should not be `now`. The APIs and other elements on top of the models should allow other services to specify it implicitly

Reply:
I did that to prevent the employees from claiming to check-in at any time they want, so the system and only the system is responsible for the integrity of the employees' check-ins and outs records.
I'll refactor it so that the user can do the full CRUD operation on the work_time.